src/main/java/com/pss/handlers/GetProjectileForce.java

```diff
@@ -0,0 +1,30 @@
+package com.pss.handlers;
+import com.pss.interfaces.*;
+
+import javax.vecmath.Vector3d;
+
+public class GetProjectileForce implements IGetProjectileForce{
```

Comment:
Can we rename this actually to something less generic like GetConstantContinualProjectileForce?

Reply:
Renamed!